src/compiler/diagnosticMessages.json

```diff
@@ -2872,26 +2872,30 @@
         "category": "Error",
         "code": 7015
     },
-    "Property '{0}' implicitly has type 'any', because its 'set' accessor lacks a type annotation.": {
+    "Property '{0}' implicitly has type 'any', because its set accessor lacks a parameter type annotation.": {
         "category": "Error",
         "code": 7016
     },
-    "Index signature of object type implicitly has an 'any' type.": {
+    "Property '{0}' implicitly has type 'any', because its get accessor lacks a return type annotation.": {

```

Comment:
@sandersn  I was doing that (i.e sharing the error message) but getter and setter actually has slightly different error (getter error because no return type annotation while setter error because no parameter type annotation) since this is to improve that I think it is worth it to separate the two.